fahrschein/src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -54,6 +58,35 @@ public static NakadiClientBuilder builder(URI baseUri) {
         }
     }
 
+    public <T> void publish(String eventName, List<T> events) throws IOException {
+        final URI uri = baseUri.resolve(String.format("/event-types/%s/events", eventName));
+        final ClientHttpRequest request = clientHttpRequestFactory.createRequest(uri, HttpMethod.POST);
+
+        request.getHeaders().setContentType(MediaType.APPLICATION_JSON);
+
+        try (final OutputStream body = request.getBody()) {
+            objectMapper.writeValue(body, events);
+        }
+
+        try (final ClientHttpResponse response = request.execute()) {
+            if ((response.getRawStatusCode() == 207 || response.getRawStatusCode() == 422)) {
```

Comment:
Since the `ClientHttpRequestFactory` can be configured by the user, there is a chance that a `ProblemHandlingClientHttpRequestFactory` isn't in place. Would that let the publishing fail silently in case of a response != `207` and `422`?